lib/facter/rh_certificates.rb

```diff
@@ -0,0 +1,36 @@
+def extract_certificates_from_subscription_manager
+  certificates = {}
+  certificate_end_path = '/cert.pem'
+  private_key_end_path = '/key.pem'
+  data = Facter::Core::Execution.exec('subscription-manager config')
+  data = data.gsub("\n", "")
+  data_array = data.scan(/(\S+)\s*=\s* ([^ ]+)/)
+  data_hash = Hash[data_array]
+  consumer_cert_dir = data_hash["consumercertdir"].gsub(/[\[\]]/, "")
```

Comment:
Do this gsub to all the values.  `subscription-manager config` surrounds default values with brackets for some reason.

```
rh_certificate_consumer_host_cert => /etc/pki/consumer/cert.pem
rh_certificate_consumer_host_key => /etc/pki/consumer/key.pem
rh_certificate_repo_ca_file => [/etc/rhsm/ca/redhat-uep.pem]
```